modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/search/test/CalendarBookingIndexerLocalizedContentTest.java

```diff
@@ -89,32 +89,18 @@ public void testJapaneseTitle() throws Exception {
 			"title_ja_JP", japaneseName
 		).build();
 
-		String word1 = "新規";
-		String word2 = "作成";
-		String prefix1 = "新";
-		String prefix2 = "作";
-
-		Stream.of(
-			word1, word2, prefix1, prefix2
-		).forEach(
-			keywords -> assertFieldValues(
-				"title", LocaleUtil.JAPAN, titleMap, keywords)
-		);
+		for (String keyword : Arrays.asList("新規", "作成", "新", "作")) {
```

Comment:
use Arrays.asList() directly while iterating. 
Also inline the word without declaring new variables